apis/v1alpha1/opentelemetrycollector_types.go

```diff
@@ -37,6 +37,10 @@ type OpenTelemetryCollectorSpec struct {
 	// +optional
 	Replicas *int32 `json:"replicas,omitempty"`
 
+	// MinReplicas sets a lower bound to the autoscaling feature
+	// +optional
+	MinReplicas int32 `json:"minReplicas,omitempty"`
```

Comment:
What is the relationship between `MinReplicas` and `Replicas`? Which one should be used when? This is something that should be documented.

Reply:
With that we probably need to add some checks to the webhook 

https://github.com/open-telemetry/opentelemetry-operator/blob/b77c89d6ea846ba5d184a4ab8716744e190ef677/apis/v1alpha1/opentelemetrycollector_webhook.go#L120-L122

to test the minreplicas or any relationship between replicas, minreplias and maxreplicas.